examples/nidaqmx/dsa-shared-timebase-and-trig-analog-input-and-output.py

```diff
@@ -0,0 +1,395 @@
+# This example configures a total of four tasks on two PXI Dynamic Signal Acquistion (DSA) devices.
+# Each device runs a task for analog input and another for analog output. This example illustrates
+# continuous analog input and output operations.
+#
+# Instructions for Running:
+#    1. Select which type of Synchronization method you want to use.
+#    2. Select the physical channel to correspond to where your
+#       signal is input on the DAQ device.
+#    3. Enter the minimum and maximum expected voltage for the input
+#       and output operations on each device.
+#    Note: For optimal accuracy, match the input range to the
+#          expected voltage level of the measured signal (for input)
+#          and generated signal (for output).
+#    4. Set the number of samples to acquire per channel. This
+#       parameter determines how many points will be read and
+#       generated with each iteration.
+#    5. Set the rate of the acquisition and generation. The same
+#       sampling rate is employed for input and output operations on
+#       each device.
+#    Note: The sampling rate should be at least 2.2 times the maximum
+#          frequency component of the signal being acquired on analog
+#          input and the signal being generated on analog output.
+#          Frequency components beyond about 0.47 times the sampling
+#          rate will be eliminated by the alias and imaging
+#          protection on the DSA device.
+#
+# Steps:
+#    1. Create a task.
+#    2. Create an analog input and output channel for both the leader
+#       and follower devices.
+#    3. Set timing parameters for a continuous acquisition. The
+#       sample rate and block size are set to the same values for
+#       each device.
+#    4. There are two types of synchronization available on DSA
+#       devices. The first method shares the Sample Clock Timebase
+#       across the PXI_Star bus. It also uses the Sync Pulse which is
+#       shared across the PXI_Trig / RTSI bus. The second method uses
+#       the Sync Pulse in conjunction with the PXI Reference clock 10
+#       on the PXI backplane.
+#    5. Call the Get Terminal Name with Device Prefix utility
+#       function. This will take a Task and a terminal and create a
+#       properly formatted device + terminal name. This signal is
+#       then used as an output generation trigger on the leader as
+#       well as and acquisition start trigger and generation start
+#       trigger on the follower. The signal is shared along the PXI_Trig
+#       / RTSI bus.
+#    6. Synthesize a standard sine waveform and load this data into
+#       the output RAM buffer for each device.
+#    7. Call the Start function to start the acquisition.
+#    8. Read all of the data continuously. The 'Samples per Channel'
+#       control will specify how many samples per channel are read
+#       each time. If either device reports an error or the user
+#       presses Enter, the acquisition will stop.
```

Comment:
PR Comment says `Ctrl+C` not Enter. but the code does look like Enter.

Reply:
It actually quits in either case. I want to document it as Enter, but I also wanted to make sure that `Ctrl+C` works to quit (which it didn't before I did some stuff)